### DISCUSSION TITLE
Edit text in Bios console

### DISCUSSION THREAD

**Discussion by tairbox** [ipxe/ipxe#513] [Q&A] [answered]
Hello!
When I type multilple strings of text in pcbios console and then i need to change something at beginning of the first line, i can't move cursor to that position. It won't move behind beginning of the current line.
Is it normal behaviour?
I want to add possibility to edit command line according [this](https://ipxe.org/cmd/read), but if i can't normally edit text in multiple strings it would be useless.

**Comment by NiKiZe** [comment, marked as answer]
Yes quite normal you can try it in the normal iPXE shell as well.
I have a feeling that a fix is more complex than what we want it to be.

**Reply by tairbox** [reply]
Thanks for the answer. In ipxe shell all works fine
